website/plugins/docusaurus-plugin-open-graph-image/layout.js

```diff
@@ -0,0 +1,30 @@
+const { createSVGText } = require("./font");
+
+function createLayoutLayers(doc, layout, previewFont, textWidthLimit) {
+    const layers = layout.map((item) => {
+        if (!Object.prototype.hasOwnProperty.call(doc, item.name)) {
+            console.error(`Wrong template config? Doc property ${item.name} not found.`);
+            return undefined;
```

Comment:
[`nitpick:`](https://conventionalcomments.org/#labels) Можно и проще, как ниже делал)

```suggestion
            return;
```

Reply:
Вернул, т.к. снова наткнулся на то что матюгальник проснулся =))
![image_2021-12-10_23-42-02](https://user-images.githubusercontent.com/1334019/145639452-f7b3ce13-d97b-4df6-853a-92865169d840.png)